test/extended/testdata/builds/test-build-app/utf8_tést_app.rb

```diff
@@ -0,0 +1,7 @@
+require 'rack'
+ 
+app = Proc.new do |env|
+    ['200', {'Content-Type' => 'text/html'}, ['Wë súpport UTF-8!']]
+end
```

Comment:
@gabemontero this dir is run in our extended `start-build` tests - see https://github.com/openshift/origin/blob/master/test/extended/builds/start.go#L172-L182

Reply:
ah ... clever turn on things ... works for me :-)